OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -27,8 +28,26 @@ class CheckTranslationReference : ILintPass
 		readonly Dictionary<string, string[]> referencedVariablesPerKey = new Dictionary<string, string[]>();
 		readonly List<string> variableReferences = new List<string>();
 
-		public void Run(Action<string> emitError, Action<string> emitWarning, ModData modData)
```

Comment:
There's an error in commits. The recent changes to CheckTranslationReference.cs where applied to the wrong commit and left me confused for about 10 minutes